drip/drips.py

```diff
@@ -116,7 +135,13 @@ def message(self):
 
             # check if there are html tags in the rendered template
             if len(self.plain) != len(self.body):
-                self._message.attach_alternative(self.body, 'text/html')
+                message_body = self.body.replace(
+                    '<body>',
+                    '<body><span style="display:none;">{}</span>'.format(
+                        self.pre_header
+                    )
+                ) if self.pre_header else self.plain
```

Comment:
this doesn't seem to be right, can you explain what are you trying to accomplish here as `pre_header` should always be in plan text?

Reply:
Yeah you right it doesn't, I was think of doing something then I changed it, but I forgot to remove this.